course/auth.py

```diff
@@ -680,9 +774,35 @@ def user_profile(request):
 
     user_form = None
 
+    def has_verified_inst_id(user):
+        if not EDITABLE_INST_ID_PRE_VRF:
+            return True and user.institutional_id or False
+        else:
+            if user.institutional_id_verified:
+                return True
+            else:
+                return False
+
+    def enable_inst_input(request):
```

Comment:
Yes. In fact, that is interface matter, just to make sure when users are login for the first time or if they are asked to update their inst_id in profile, all the fields are shown, if he/she is allow to modify those fields. This just skips a step for the user to uncheck the ``I have no inst_id`` checkbox. 

Do you suggest I should let the js to do the trick rather than in python backend?